lightstep/resource_alert_test.go

```diff
@@ -17,7 +17,7 @@ func TestAccAlert(t *testing.T) {
 
 	badAlertMissingQueryAndCompositeFields := `
 resource "lightstep_alert" "errors" {
-  project_name = "terraform-provider-tests"
+  project_name = ` + fmt.Sprintf("\"%s\"", testProject) + `
```

Comment:
I know this is just a test, but given this is repeated many times, would it make sense to simplify all these `fmt.Sprintf` calls...

From:

```go
project_name = ` + fmt.Sprintf("\"%s\"", testProject) + `
```

To:

```go
project_name = "` + testProject + `"
```

...or maybe move it to a common variable in some form?

Reply:
Yup, this was me being lazy because this was already done in some stream alerts, so I just copied what was done there, but update it with your suggestion!